### PR TITLE
Fixes #19365 - display loading spinner instead of text

### PR DESCRIPTION
**Pull request by lizagilman**

### CODE REVIEW

File: app/assets/javascripts/charts.js [hunk 1]
```diff
@@ -297,7 +297,7 @@ function get_pie_chart(div, url) {
     $('body').append('<div id="' + div + '" class="modal fade"><div class="modal-dialog"><div class="modal-content"></div></div></div>');
     $("#"+div+" .modal-content").append('<div class="modal-header"><button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button><h4 class="modal-title"></h4></div>')
         .append('<div class="container">')
-        .append('<div id="' + div + '-body" class="fact_chart modal-body">' + __('Loading') + ' ... </div>')
+        .append('<div id="' + div + '-body" class="modal-body">' + '<div class="spinner spinner-lg modal-spinner"></div>' + ' </div>')
```
Comment (tbrisker) [inline app/assets/javascripts/charts.js]: no need to separate this into another string, this was originally done because `Loading` needs translation

Comment (lizagilman) [reply app/assets/javascripts/charts.js]: updated

**Comment by tbrisker** [comment]
Works well, merging. Eventually we should migrate this to use react charts.

**Comment by tbrisker** [comment]
Thanks @lizagilman !

**Pull request merged**